packages/contracts/src/utils/token_registry_wrapper.ts

```diff
@@ -4,13 +4,16 @@ import { TokenRegistryContract } from '../contract_wrappers/generated/token_regi
 
 import { Token } from './types';
 
+import { constants } from './constants';
+import { web3Wrapper } from './web3_wrapper';
```

Comment:
Can we pass in a provider and instantiate a new web3Wrapper instance here like we do in the other wrappers?

Reply:
👍